modules/ROOT/pages/6.1-release-notes.adoc

```diff
@@ -0,0 +1,1121 @@
+	  editor.ui.registry.addSidebar('showcomments', {
+		tooltip: 'showcomments',
+		icon: 'comment',
+		onSetup: (api) => {
+		  console.log('Render panel', api.element());
+		},
+		onShow: (api) => {
+		  console.log('Show panel', api.element());
+		},
+		onHide: (api) => {
+		  console.log('Hide panel', api.element());
+		}
+	  });
+	}
+----
+
+NOTE: The comment sidebar presents with no animation or other effect. It is just there, along with any other initialised {productname} editor components. Also, as noted above, the sidebar does not take focus, which remains with the main editor.
+
+
+==== Support annotating certain block elements directly when using the editor annotator API
+
+Several new block elements can now have comments attached to them, including:
+
+* Tables of contents
+* Images
+* Images with captions
+* Code samples
+* media in iframes
+* embedded media
+* embedded pages
+
+When comments are showing, a 3px wide yellow (hex color #FFE89D) rectangle presents around any element with a comment attached.
+
+NOTE: Enablng this feature depends on changes made to the Core {productname} editor. As a consequence, adding comments to these elements requires both {productname} 6.1 or later and version 3.1.0 or later of the xref:comments.adoc[Comments Premium Plugin].
+
+
+==== The label name for the Save Comment button can now be customized
+
+Previously, the Save Comment was labeled *Save*. After field reports that end-users were interpreting this to mean the entire {productname} document would be saved on pressing this button, two changes have been made.
+
+1. The default label for this button is now *Comment*.
++
+This new default presents for both new comments and replies to existing comments.
+
+2. This default label is now customisable.
++
+_How does one customise this? I presume it’s a value one sets as part of `tinymce.init` but which one? And where in this configuration file?_
++
+_Or is it a matter of an integrator editing the `.pot` file — `src/text/pot/plugins/tinycomments/tinymce_tinycomments.pot` — that comes along for the ride?_
+
+NOTE: when a comment is being edited, the default button, which writes the changes back and closes the Edit Comment dialog, is still labeled *Save*.
+
+
+For information on the **Comments** plugin, see: xref:comments.adoc[Comments plugin].
+
+
+=== Enhanced Media Embed 3.0.1
+
+The {productname} vnumtxt release includes an accompanying release of the **Enhanced Media Embed** premium plugin.
+
+**Enhanced Media Embed** 3.0.1 fixes the following bug:
+
+==== Enhanced Media Embed tests failed in several browsers
+
+An incorrect `RealKeys()` setting caused the Enhanced Media Embed webdriver tests to fail in several browsers (including Firefox, Microsoft Edge, and Safari).
+
+With this update, this setting has been corrected, from `RealKeys.text(\n)` to `RealKeys.combo({}, 'enter')`. As a consequence, the Enhanced Media Embed webdriver tests now pass in all supported browsers, as expected.
```

Comment:
I don't know why this went into the changelog. Customers do not have access to the source code or tests of our premium plugins, so it makes no sense to talk about a test fix in the release notes.

- [ ] Remove enhanced media embed section, or perhaps leave a note that the version bump was to fix internal technical issues only?

Reply:
Yeah this looks like the wrong details were pulled... the fix for Enhanced Media Embed was:
```
- Console warnings would incorrectly be logged when not using the media plugin #TINY-7375
```

Reply:
Keenly aware how passive-aggressive this sounds, but if you read the description and note for [TINY-7375](https://ephocks.atlassian.net/browse/TINY-7375), you’d be forgiven for presuming the fix concerned test failures.

That said, the *Enhanced Media Embed 3.0.1* section has been re-written.

Reply:
Yeah, this was just some found while fixing the failing tests issue. It possibly should have been a separate JIRA, but this is also why it's important to look at the actual changelog I guess 🤷‍♀️